AB_environments/README.md

```diff
@@ -46,6 +46,23 @@ forks, e.g.
       - dask ==2023.4.2
       - git+https://github.com/yourname/distributed@803c624fcef99e3b6f3f1c5bce61a2fb4c9a1717
 ```
+
+You may also ignore the recipe file and go for a barebones environment. The bare
```

Comment:
Can you elaborate why one might want to do that?

Reply:
For example because you want to see how pandas performs without pyarrow (fall back to fastparquet), how xarray performs without h5netcdf (fall back to scipy), etc